tasks/main.yml

```diff
@@ -17,6 +17,23 @@
     group: 'nginx'
     mode: 0775
 
+- name: Rotation for project nginx log file
+  logrotate:
+    name: '{{ application_name }}'
+    path: '/var/log/nginx/{{ application_name }}/*.log'
+  args:
+    options:
+      - 'daily'
+      - 'rotate 14'
```

Comment:
Sorry, I just checked the web servers. We have 30 days for juwai.com. Should probably keep the same. Maybe it's best to create a variable for this.